languages.toml

```diff
@@ -1321,12 +1321,13 @@ source = { git = "https://github.com/DerekStride/tree-sitter-sql", rev = "0caa7f
 name = "gdscript"
 scope = "source.gdscript"
 injection-regex = "gdscript"
-file-types = ["gd"]
-shebangs = []
+file-types = ["gd", "gdscript", "gdscript3", "gdscript4"]
 roots = ["project.godot"]
 auto-format = true
 comment-token = "#"
-indent = { tab-width = 4, unit = "    " }
+indent = { tab-width = 4, unit = "	" }
+# Change this to port 6005 once Godot 4 is released.
+language-server = { command = "nc", args = ["localhost", "6008"] }
```

Comment:
I'm not sure we should use `nc` for the language server in the main `languages.toml` config. `nc` is just a hack around the language server not supporting stdio and I'm not sure windows has `nc` by default. Instead I think this configuration should be listed in https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers

Reply:
Well, there is a note about netcat at the top of the wiki page. Should this be closed then?

Reply:
The file-types and indent changes are still good so I wouldn't say this should be closed. Actually we may want to merge this as-is since - if I understand correctly - the godot LSP is built into the godot editor so there's no chance of being able to run by stdio unless godot decides to split out the LSP into its own executable/project. What do you think @archseer?